src/browser/core_state.ts

```diff
@@ -401,15 +399,15 @@ export function getMainWindowOptions(id: number): Shapes.WindowOptions|void {
 
     if (!app) {
         console.warn('getMainWindowOptions - app not found', arguments);
-        return; //throw new Error('getMainWindowOptions - app not found');
+        return;
     }
 
     if (!app.appObj) {
         console.warn('getMainWindowOptions - app opts not found', arguments);
-        return; //throw new Error('getMainWindowOptions - app opts not found');
+        return;
     }
 
-    // console.log('getMainWindowOptions', app.appObj._options);
+
```

Comment:
"...Really makes the code breathe..." ;)

Reply:
👍 will update